pkg/cmd/streamsGet.go

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package cmd
+
+import (
+	"log"
+
+	"github.com/galasa-dev/cli/pkg/api"
+	"github.com/galasa-dev/cli/pkg/galasaapi"
+	"github.com/galasa-dev/cli/pkg/spi"
+	"github.com/galasa-dev/cli/pkg/streams"
+	"github.com/galasa-dev/cli/pkg/utils"
+	"github.com/spf13/cobra"
+)
+
+// Objective: Allow user to do this:
+//
+//	streams get
+type StreamsGetCommand struct {
+	cobraCommand *cobra.Command
+}
+
+// ------------------------------------------------------------------------------------------------
+// Constructors methods
+// ------------------------------------------------------------------------------------------------
+func NewStreamsGetCommand(
+	factory spi.Factory,
+	streamsGetCommand spi.GalasaCommand,
+	commsFlagSet GalasaFlagSet,
+) (spi.GalasaCommand, error) {
+
+	cmd := new(StreamsGetCommand)
+	err := cmd.init(factory, streamsGetCommand, commsFlagSet)
+	return cmd, err
+
+}
+
+// ------------------------------------------------------------------------------------------------
+// Public methods
+// ------------------------------------------------------------------------------------------------
+func (cmd *StreamsGetCommand) Name() string {
+	return COMMAND_NAME_STREAMS_GET
+}
+
+func (cmd *StreamsGetCommand) CobraCommand() *cobra.Command {
+	return cmd.cobraCommand
+}
+
+func (cmd *StreamsGetCommand) Values() interface{} {
+	return nil
+}
+
+// ------------------------------------------------------------------------------------------------
+// Private methods
+// ------------------------------------------------------------------------------------------------
+func (cmd *StreamsGetCommand) init(factory spi.Factory, streamsCommand spi.GalasaCommand, commsFlagSet GalasaFlagSet) error {
+
+	var err error
+	cmd.cobraCommand, err = cmd.createCobraCmd(factory, streamsCommand, commsFlagSet)
+	return err
+
+}
+
+func (cmd *StreamsGetCommand) createCobraCmd(
+	factory spi.Factory,
+	streamsCommand spi.GalasaCommand,
+	commsFlagSet GalasaFlagSet,
+) (*cobra.Command, error) {
+
+	var err error
+
+	commsFlagSetValues := commsFlagSet.Values().(*CommsFlagSetValues)
+	streamCommandValues := streamsCommand.Values().(*StreamsCmdValues)
+
+	streamsGetCobraCmd := &cobra.Command{
+		Use:     "get",
+		Short:   "Gets a list of test streams",
+		Long:    "Get a list of test streams stored in the Galasa API server",
+		Aliases: []string{COMMAND_NAME_STREAMS_GET},
+		RunE: func(cobraCommand *cobra.Command, args []string) error {
+			return cmd.executeStreamsGet(
+				factory, streamsCommand.Values().(*StreamsCmdValues), commsFlagSetValues,
+			)
+		},
+	}
```

Comment:
The `--format` flag isn't being added? Looks like it's not been implemented yet?